app/src/main/java/org/mifos/mobilebanking/utils/MFDatePicker.java

```diff
@@ -91,5 +102,9 @@ public void setOnDatePickListener(OnDatePickListener onDatePickListener) {
         public void onDatePicked(String date);
     }
 
+    public boolean isStartDate(boolean isStart) {
```

Comment:
As you are using this function to just update the isStart Value so you can change the return type to void and Remove the return statement.